### ISSUE TITLE
VMMDLL_PROCESS_INFORMATION has no member os

### ISSUE THREAD

**Issue by slack69** [ufrisk/pcileech#119]
Not sure why but in `umd.c` you seem to try and access `os` which is not a member of `VMMDLL_PROCESS_INFORMATION`:

https://github.com/ufrisk/pcileech/blob/fdbf2d7f06d9afd7e47b32981e28b87cdb259ef9/pcileech/umd.c#L42

This stopped me from compiling until I commented out the printf statement.

**Comment by ufrisk** [comment]
Thanks. I messed up somewhat. I updated the vmmdll.h file in my MemProcFS project and I just copied the new version into the PCILeech project, and it will break on the syntax right now.

I have it fixed in the next release I'm planning probably sometimes in May. Right now it should be easy to just remove `os.` and compile. Nothing have changed with regards to struct positions and such, I just made a simplification to the internal naming in it.
